lib/decision_review_v1/service.rb

```diff
@@ -226,11 +226,11 @@ def get_notice_of_disagreement_upload(guid:)
       end
     end
 
-    def self.file_upload_metadata(user)
+    def self.file_upload_metadata(user, backup_zip = nil)
       {
         'veteranFirstName' => transliterate_name(user.first_name),
         'veteranLastName' => transliterate_name(user.last_name),
-        'zipCode' => user.postal_code,
+        'zipCode' => user.postal_code || backup_zip,
```

Comment:
Is it possible `backup_zip` could ever be `nil`?

Reply:
IE ``` nil || nil => nil```, so it should be fine.